firmware/cmake/generate_cubemx.mk

```diff
@@ -14,17 +14,20 @@ CUBEMX_GEN_SCRIPT = cubemx_script.txt
 # Makefile, i.e. whenever IOC_FILE has been updated but new code has not been
 # generated.
 
-Makefile: $(IOC_FILE)
-# Create an file containing commands to generate the cubemx code.
-	echo "config load \"$(IOC_FILE)\"" > $(CUBEMX_GEN_SCRIPT)
-	echo "project generate ./" >> $(CUBEMX_GEN_SCRIPT)
-	echo "exit" >> $(CUBEMX_GEN_SCRIPT)
+.PHONY: Makefile
```

Comment:
This should not be PHONY: Makefile is a real file target

Reply:
Given we are not producing the file and simply modifying it, would we consider that to be a real file target?

Reply:
Getting solved in #111